packages/dx-react-grid/src/plugins/table-edit-column.test.jsx

```diff
@@ -175,13 +168,13 @@ describe('TableHeaderRow', () => {
           ...defaultDeps.template.tableCell,
           row: defaultDeps.template.tableCell.tableRow.row,
         });
-      expect(tree.find(defaultCommandComponents.edit).exists())
+      expect(tree.find(defaultProps.commandComponent).filterWhere(wrapper => wrapper.prop('id') === 'edit').exists())
```

Comment:
Too long string `defaultProps.commandComponent).filterWhere(wrapper => wrapper.prop('id') `. Can we extract it?